interpreter/interpretable.go

```diff
@@ -873,6 +1028,27 @@ func (fold *evalExhaustiveFold) Eval(ctx Activation) ref.Val {
 	return res
 }
 
+// Cost implements the Coster interface method.
+func (fold *evalExhaustiveFold) Cost() (min, max int64) {
```

Comment:
Make sure you document the logic for how cost is computed for the fold, can be internal to the function here given the complexity.

Reply:
done